src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -651,6 +652,17 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 		}
 	}
 
+	private func setupAntigenTestProfileForTesting() {
+		if let antigenTestProfileInfoScreenShown = UserDefaults.standard.string(forKey: "antigenTestProfileInfoScreenShown") {
+			store.antigenTestProfileInfoScreenShown = antigenTestProfileInfoScreenShown != "NO"
+		}
+		if let removeAntigenTestProfile = UserDefaults.standard.string(forKey: "removeAntigenTestProfile") {
+			if removeAntigenTestProfile == "YES" {
+				store.antigenTestProfile = nil
+			}
+		}
```

Comment:
why no bool?
```suggestion
		if UserDefaults.standard.bool(forKey: "antigenTestProfileInfoScreenShown") {
			store.antigenTestProfileInfoScreenShown.setValue(false, forKey: "antigenTestProfileInfoScreenShown")
		}
		if UserDefaults.standard.bool(forKey: "removeAntigenTestProfile") {
			store.antigenTestProfile = nil
		}
```

Reply:
I copied code from other implementations. Can you pass a bool value to the launch arguments?

Reply:
At the moment no, so having strings are fine from my POV.

Reply:
@whiskey I seems we have to do it this way.

Reply:
@freshking @naveeddotio You CAN!

<img width="151" alt="Screen Shot 2021-05-07 at 12 16 25" src="https://user-images.githubusercontent.com/149976/117435587-4ec7b780-af2e-11eb-910f-813a1c747fc5.png">
<img width="613" alt="Screen Shot 2021-05-07 at 12 16 41" src="https://user-images.githubusercontent.com/149976/117435582-4cfdf400-af2e-11eb-8d05-109b8eba6326.png">
(tested with my app which doesn't take ages to start)

Reply:
Honestly, I tried this with Bool and it didn't work for me. So, I looked in to the app and it was string everywhere, so I changed it to string. If it works for you Bastian, give it a try.

Reply:
Have you tried the "good" old Obj-C bools? Not that fancy new Swift shit you young kids play the whole time! 👨🏻‍🦳

😉